server/src/main/java/org/candlepin/audit/EventMessageReceiver.java

```diff
@@ -71,6 +61,8 @@ public void onMessage(ClientMessage msg) {
             String reason = (e.getCause() == null) ? e.getMessage() : e.getCause().getMessage();
             log.error("Unable to process message {}: {}", messageId, reason);
 
+            log.debug("Message listener {} processed message: {}:FAILURE", listener, msg.getMessageID());
```

Comment:
If there's another change to be made before this gets merged, can we also add a space between the colon and the SUCCESS/FAILURE of these messages? If nothing else ends up being done before merging, we can disregard it.

Reply:
Will do this when I fix the test.